<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filtro de im√°genes</string>
  </property>
  <property name="windowOpacity">
   <double>5.000000000000000</double>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Presione para cargar una imagen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Presione para cargar una imagen</string>
    </property>
    <property name="text">
     <string>Cargar imagen</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelImagen1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>561</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="paralelismo">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>Seleccione el porcentaje de paralelismo</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Porcentaje de paralelismo NIOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>560</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>560</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelImagen2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>561</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPorcentaje">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>10</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>10</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonIniciar">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
